CHANGELOG.md

```diff
@@ -1,8 +1,25 @@
-# dbt_snapchat_ads version.version
+# dbt_snapchat_ads v0.8.0
+This release introduces the following updates:
+
+## Breaking Changes
+- Adds ad squad and campaign details to the `snapchat_ads__ad_report` model, so that they can eventually be incorporated in the `dbt_ad_reporting` package for the `ad_reporting__ad_report` model.  ([#33](https://github.com/fivetran/dbt_snapchat_ads/pull/33))
+- Fields added include:
+  - `ad_squad_id`
+  - `ad_squad_name`
+  - `campaign_id`
+  - `campaign_name`
```

Comment:
Can you consolidate this into the one bullet above. "Fields added include:" doesn't need to be a separate bullet. But I would like the separate fields to be their own sub bullets still.